tests/contrib/scrapy/test_pipeline_cerberus.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import
+from slugify import slugify
+from scrapy.utils.test import get_crawler
+from scrapy import Item
+from functools import partial
+import sys
+
+import pytest
+from spidermon.contrib.scrapy.pipelines import ItemValidationPipeline
+from tests.fixtures.items import TestItem, TreeItem
+from tests.fixtures.validators import cerberus_tree_schema, cerberus_test_schema
+
+STATS_AMOUNTS = "spidermon/validation/validators"
+STATS_ITEM_ERRORS = "spidermon/validation/items/errors"
+STATS_MISSINGS = "spidermon/validation/fields/errors/missing_required_field"
+STATS_TYPES = "spidermon/validation/validators/{}/{}"
+
+SETTING_CERBERUS = "SPIDERMON_VALIDATION_CERBERUS"
+
+def assert_type_in_stats(validator_type, obj):
+    return "'{}' in {{stats}}".format(
+        STATS_TYPES.format(obj.__name__.lower(), validator_type)
+    )
+
+ASSERT_TYPE_IN_STATS = partial(assert_type_in_stats, "cerberus")
```

Comment:
Does it really need to be a `partial`?
Wouldn't be easy to assume `cerberus` in `assert_type_in_stats`?

Reply:
Well, this was part of my brainstorming questions that we needed to discuss but didn't ask.

`partial` has been used in `test_pipelines.py` to kinda delay the first argument's input just because in one class the `validator_type` is  `jsonschema` and the other its `schematics`.

My question here was:
I was earlier thinking of converting this to a fixture. What say about that? It seemed like overkill hence didn't do it. Later I thought about hardcoding which is clean but then we needed the obj so functions are a way to go.   
Since for this specific test. Cerberus will always be there hence any of the options seemed like an okay choice. Hence, I stuck with `partial` just because that's what the other tests did. 

Ps. Partial is pretty neat only when used in the right place. Didn't know about this.
Can transform this into a function? Let me know.

Reply:
For me, since we're only dealing with Cerberus, there is no need for an argument.
So, there is no need for `partial`;
Also, a `fixture` for this does not make sense, `fixtures` are for test inputs/state, not assertions.

Reply:
It is completely fine for a function to be `assert_type_in_stats` and set cerberus as constant here.
At least, also, when using `partial` is not a case to set the function `UPPER_CASE` like a constant, it is just a function. So, a better name would be `assert_in_cerberus_stats` for example

Reply:
Done and roger that! Good change, with the naming conventions!

Reply:
I keep following Pylint, now that I think of it. Maybe following too much of set of conventions can be counterproductive.

Reply:
Done! :heavy_check_mark: